libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/hwdef.dat

```diff
@@ -32,7 +32,7 @@ define HAL_STORAGE_SIZE 16384
 I2C_ORDER I2C2 I2C1
 
 # order of UARTs (and USB)
-UART_ORDER OTG1 UART4 USART2 USART3 UART8 UART7
+UART_ORDER OTG1 UART4 USART2 USART3 UART8 UART7 OTG2
```

Comment:
we really need to switch hal.uartN to being an array ....
this change could put OTG2 past the last hal.uartN driver on some boards

Reply:
Ok, will look into that.